bugfix: S3C-3830 bump warp10

rahulreddy opened the pull request.

rahulreddy committed:
bugfix: S3C-3830 bump warp10

bert-e commented:
# Hello rahulreddy,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/utapi/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/S3C-3830-fetch-warp10-results` into
`development/7.9`
* [w/8.1/bugfix/S3C-3830-fetch-warp10-results](https://github.com/scality/utapi/compare/development/8.1...w/8.1/bugfix/S3C-3830-fetch-warp10-results) will be merged into `development/8.1` (pull request #897)



The following branches will **NOT** be impacted:

* `development/6.4`
* `development/7.4`




*Follow* integration pull requests if you would like to be notified of
build statuses by email.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers





Peer approvals *must* include a mandatory approval from @tmacro.

tmacro committed:
update method for seed generation, remove unused initial token gen

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

tmacro commented:
Upstream has removed the python script that generates the random seeds during initial startup, and replaced it with a java implementation. I've updated `10-generate-hash-seeds` to use the new one.
I've also taken the opportunity to remove `40-generate-secrets` since we use a static token and I suspect a few errors seen in the field to be cause by some flakiness here.
Finally I've added `ENV S6_BEHAVIOUR_IF_STAGE2_FAILS 2` which will cause s6 to exit on any failure during `cont-init` rather than continuing with startup.

rahulreddy commented:
> Upstream has removed the python script that generates the random seeds during initial startup, and replaced it with a java implementation. I've updated `10-generate-hash-seeds` to use the new one.
> I've also taken the opportunity to remove `40-generate-secrets` since we use a static token and I suspect a few errors seen in the field to be cause by some flakiness here.
> Finally I've added `ENV S6_BEHAVIOUR_IF_STAGE2_FAILS 2` which will cause s6 to exit on any failure during `cont-init` rather than continuing with startup.

How do you usually track/figure out these changes?

rahulreddy commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/7.9`

* :heavy_check_mark: `development/8.1`



The following branches will **NOT** be impacted:

* `development/6.4`
* `development/7.4`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/7.9`

* :heavy_check_mark: `development/8.1`



The following branches have **NOT** changed:

* `development/6.4`
* `development/7.4`



Please check the status of the associated issue S3C-3830.

Goodbye rahulreddy.
